crates/telio-traversal/src/connectivity_check/cross_ping_check.rs

```diff
@@ -220,6 +228,34 @@ impl CrossPingCheck {
 
 #[async_trait]
 impl CrossPingCheckTrait for CrossPingCheck {
+    async fn pause(&self, key: &PublicKey) -> Result<(), Error> {
+        telio_log_debug!("pause for {}", key);
```

Comment:
Would it make sense to add a bit more info to this line (and the same line for `unpause`) so it's more obvious from looking at the logs what it means? Maybe something like `Pausing CPC for {key}` would be sufficiently descriptive

Reply:
I like the suggestion, did that.
Though using telio macros alerady emit the component name in front, like such:
```
2024-10-03 09:04:24.628531 TelioLogLevel.DEBUG "telio_traversal::connectivity_check::cross_ping_check":936 message...
```

so I tried to follow the other logs and not include the name :)